### PR TITLE
Adds new "Video Available on Web Only" setting

### PR DESCRIPTION
**Pull request by lamagnifica**
see lines 321-324.
@gwprice , @catong , @lou-wang , @jhendersonedx 
Fixes DOC-1765

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst [hunk 1]
```diff
@@ -9,8 +9,8 @@ Introduction to Videos
 **********************
 
 You can create a video of your lecture and add it to your course with other
-components—such as discussions and problems—to promote active learning. Adding
-a video to your course has several steps.
+components, such as discussions and problems, to promote active learning.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: The wording of "..add it to your course with other components" is hard to parse and unnecessarily confusing  -- it's as if we are talking about adding the other types of components at the same time as the video. If we want to mention the other components I'd recommend rewording... talk about adding video to _supplement_ other components in your course.

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst [hunk 2]
```diff
@@ -200,12 +200,12 @@ Step 4. Create a Video Component
 
    You replace the default values with your own. 
    
-3. In the **Component Display Name** field, enter the name you want students to
-   see when they hover the mouse over the unit in the course ribbon. This text
-   also appears as a header for the video.
+3. In the **Component Display Name** field, enter an identifying name for this
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst:203]: How about "enter a name to identify this video."

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst [hunk 3]
```diff
@@ -220,13 +220,13 @@ Step 4. Create a Video Component
...
+    student's computer plays.
 
 5. Next to **Default Timed Transcript**, select an option. 
 
-   * If edX already has a transcript for your video--for example, if you are
-     using a video from an existing course--Studio automatically finds the
-     transcript and associates the transcript with the video.
+   * If edX already has a transcript for this video, Studio automatically
+     finds the transcript and associates the transcript with the video. This
+     typically occurs when you are reusing a video from an existing course.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: "This ^situation^ typically occurs..."

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst [hunk 4]
```diff
@@ -318,6 +318,10 @@ The following options appear on the **Advanced** tab in the video component.
       - Allows you to upload a handout to accompany this video. Your handout
         can be in any format. Students can download the handout by clicking
         **Download Handout** under the video.
+    * - **Video Available on Web Only**
+      - If True, indicates that students are only allowed to play this video
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: Do we say "If True" everywhere else? Typically I prefer "If the value of this setting is True"... more verbose but better for translation.

Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: Not sure if there is a nuance to "are only allowed to play". How about "can only" instead, if there's really no permission-related issue.

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst [hunk 5]
```diff
@@ -318,6 +318,10 @@ The following options appear on the **Advanced** tab in the video component.
...
+        in a Web browser. If False, students can use any compatible
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: Same as comment above: "If the value of this setting is False" (if not inconsistent with other occurrences in this file)

**Comment by catong** [comment]
@lamagnifica Looks good; :+1: with some suggestions.

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst:203]: the "enter an identifying name" phrasing was recently added to all of the files in the developing_course directory. I'd rather stick with the same phrasing for components as well, although it is still too soon to say that the name of a non-problem will be surfaced in Insights...

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: It is a legal requirement, often because a video contains material that is being used by permission, and that permission only grants them the right to distribute through the edX web site. But, since this is for open source too, we can't use that as an example...

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst (hunk 5 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: This file does not currently tell you what happens if you set any of the other  advanced settings to true or to false. I would like to move in that direction, as I think it alleviates the, well, which one do I set it to then? response to a description. It is not my intention to research and rewrite all of these settings for this PR however....

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: I think the phrasing, "If the value of this setting is True..." assumes that you are reviewing an existing setting for reference, rather than actively selecting a setting. As an alternative to the "If True/If False" construction, maybe "If {whatever}, select True."?

Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: I like it! "If (you) select True..."

**Comment by gwprice** [comment]
:+1:
